common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -25,6 +25,71 @@ steps:
       scriptPath: common/scripts/run_rush_change.py
       arguments: $(Build.SourceBranch) $(Build.Reason) $(System.PullRequest.TargetBranch)
       failOnStderr: true
+
+  #### TEMP DO NOT MERGE
+
+  # DEPS
+
+  - script: sudo apt-get install unzip
```

Comment:
do not resolve this comment until this has been removed. This is how we use the custom native addon.

Once this PR and the corresponding native PR is approved I will merge the native PR, queue a new native addon, remove this section, make sure CI passes under the new addon, and then merge this.

Please comment otherwise. Also @DanRod1999 please comment if I can wait for the new imodel02 workflow instead.

also note that this section would be much smaller, but nick's template is awaiting some changes to make it more cross platform, so I inlined it here with those changes.

Reply:
Also note that the extract-api pipeline will fail until the corresponding new addon is released. iirc it's a github action and so I can't fix it the way I did the main CI jobs

Reply:
unzip should now be installed in the imodeljs Linux pool. Don't wait for the new workflow, still figuring out what that will look like. I need to discuss with @calebmshafer.

Reply:
I thought nick's PR (linked in my first comment) already makes it cross platform?

Reply:
nevermind this, doing the old style imodel02 workflow, see other comments